hail/python/hailtop/hailctl/dev/config/set_property.py

```diff
@@ -0,0 +1,27 @@
+import os
+import json
+
+from hailtop.config import get_deploy_config
+
+
+def init_parser(parser):
+    parser.add_argument("property", type=str,
+                        help="Property to set.",
+                        choices=['location', 'default', 'domain'])
+    parser.add_argument("value", type=str,
+                        help="Value to set property to.")
+
+
+def main(args):
+    deploy_config = get_deploy_config()
+    config = deploy_config.get_config()
+
+    p = args.property
+    if p == 'default':
+        p = 'default_namespace'
```

Comment:
I don't like this, it's a confusing name and this is extra complexity. I'm pretty sure all of us have canned scripts / history lines for `hailctl dev config` anyway, let's go with `hailctl dev config set default_namespace ...`.

Reply:
Agreed, fixed.